docs/gateway.md

```diff
@@ -70,3 +70,10 @@ here the backend is the service `httpecho` in the `trigger` namespace listening
 **_NOTE:_**  This is only needed for phase 1. This component will get removed when we move to phase 2 and have the event listeners provisioned through kcp.
 
 ---
+
```

Comment:
We also need to update content under Configuration section.
eg: server s1 httpecho.trigger.svc.cluster.local:80 -> server el-pac el-pipelines-as-code-interceptor.openshift-pipelines.svc.cluster.local:8080

Reply:
done